src/PowerShellEditorServices/Analysis/AnalysisService.cs

```diff
@@ -479,10 +526,12 @@ private PSObject[] InvokePowerShell(string command, IDictionary<string, object>
                     powerShell.AddParameter(kvp.Key, kvp.Value);
                 }
 
-                var result = new PSObject[0];
+                PowerShellResult result = null;
```

Comment:
Does this need to be set to null? Just want to make sure we don't cause a null pointer exception somewhere else :)

Reply:
I think it has to be set to something here, and creating a new PowerShell result would require creating two new arrays as well. Doesn't seem right.

I think I've dealt with the possibility that any result here could return `null` at the call site.

**But**, perhaps it's better to do something here like throw an analysis exception, or something else we catch??